test/Unit/Durable/DurableTimerTests.cs

```diff
@@ -0,0 +1,260 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Test.Durable
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Management.Automation;
+    using System.Threading;
+    using Microsoft.Azure.Functions.PowerShellWorker.Durable;
+    using Moq;
+    using Xunit;
+
+    public class DurableTimerTests
+    {
+        private const DateTimeKind utc = DateTimeKind.Utc;
+        private DateTime time = new DateTime(2020, 1, 1, 0, 0, 0, 0, utc);
+        private int longIntervalSeconds = 5;
```

Comment:
`const`?

Reply:
Naming convention for constants: the first letter is uppercased, e.g. `LongIntervalSeconds`.